packages/console/src/pages/Applications/index.tsx

```diff
@@ -19,23 +28,35 @@ const Applications = () => {
       <table className={styles.table}>
         <thead>
           <tr>
-            <td className={styles.applicationName}>Application Name</td>
-            <td>Client ID</td>
+            <td className={styles.applicationName}>{t('applications.application_name')}</td>
+            <td>{t('applications.application_name')}</td>
```

Comment:
The second column title should be `Client ID`, right?

Reply:
updated